cos-fleetshard-operator-camel/pom.xml

```diff
@@ -21,13 +21,22 @@
         <quarkus.kubernetes.app-config-map>${project.artifactId}-config</quarkus.kubernetes.app-config-map>
         <quarkus.kubernetes.add-build-timestamp>false</quarkus.kubernetes.add-build-timestamp>
         <quarkus.openshift.add-build-timestamp>false</quarkus.openshift.add-build-timestamp>
+        <override-app-config.name>${quarkus.kubernetes.app-config-map}-override</override-app-config.name>
 
         <!-- operator sdk -->
         <quarkus.operator-sdk.crd.generate>false</quarkus.operator-sdk.crd.generate>
         <quarkus.operator-sdk.crd.validate>false</quarkus.operator-sdk.crd.validate>
         <quarkus.operator-sdk.crd.apply>false</quarkus.operator-sdk.crd.apply>
         <quarkus.operator-sdk.disable-rbac-generation>true</quarkus.operator-sdk.disable-rbac-generation>
 
+        <!-- config map -->
+        <quarkus.kubernetes.config-map-volumes.app-config-override.config-map-name>${override-app-config.name}</quarkus.kubernetes.config-map-volumes.app-config-override.config-map-name>
+        <quarkus.kubernetes.config-map-volumes.app-config-override.default-mode>0644</quarkus.kubernetes.config-map-volumes.app-config-override.default-mode>
+        <quarkus.kubernetes.config-map-volumes.app-config-override.optional>true</quarkus.kubernetes.config-map-volumes.app-config-override.optional>
+        <quarkus.kubernetes.mounts.app-config-override.path>/mnt/app-config/override.properties</quarkus.kubernetes.mounts.app-config-override.path>
+        <!-- enable config overlay -->
+        <quarkus.kubernetes.env.vars.OVERRIDE_PROPERTIES_LOCATION>/mnt/app-config/override.properties/override.properties</quarkus.kubernetes.env.vars.OVERRIDE_PROPERTIES_LOCATION>
```

Comment:
Is this location correct?

Reply:
I think so, yes. What are your doubts? should have been without the filename?

Reply:
Is it really a directory named `override.properties` followed by a file named `override.properties`?

Reply:
Yes, ask k8s why...